scenariolib/event_search.go

```diff
@@ -12,20 +12,19 @@ import (
 
 // SearchEvent a struct representing a search, is defined by a query to execute
 type SearchEvent struct {
-	query string
-	// keyword exists because the query sent to the index may be different than the keyword(s) used to search
+	query       string
+	// keyword exists because the query sent to the index may be diffrent than the keyword(s) used to search
 	keyword     string
 	actionCause string
 	actionType  string
 	logEvent    bool
 	customData  map[string]interface{}
 }
 
-func newSearchEvent(e *JSONEvent, c *Config) (*SearchEvent, error) {
+func newSearchEvent(e *JSONEvent, c *Config, v *Visit) (*SearchEvent, error) {
```

Comment:
I would prefer if never both visit and config were passed as params. If you only need the language of the visit, only receive the language, there is also a pointer to the current config in the visit if you really need it.